<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(31, 13, 14);
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 6px 8px;</string>
    </property>
    <property name="text">
     <string>Book Count :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BookList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 13, 14);
color: rgb(255, 255, 255);
font-weight:700;
font-size:40px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Library Management System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 6px 8px;</string>
    </property>
    <property name="text">
     <string>Book List :</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid black;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="bookCount">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
qproperty-alignment: 'AlignCenter';</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>361</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid black;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 6px 8px;
border:none</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Create User&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 6px 8px;
border:none</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addUserButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: rgb(85, 255, 0);
color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="addUserInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="detailsButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="assignBookButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Assign</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bookReturnButtton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: #fdffcd;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Book Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookDetailsButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius:15px;
background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Book Details</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>label_3</zorder>
   <zorder>updateButton</zorder>
   <zorder>BookList</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>detailsButton</zorder>
   <zorder>assignBookButton</zorder>
   <zorder>bookReturnButtton</zorder>
   <zorder>BookDetailsButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
